src/utils/validations/validations.js

```diff
@@ -40,7 +40,7 @@ export const nameValidator = (value, constants) => {
 	}
 	if (isEmpty(value)) {
 		message = '';
-	} else if (!nameRegExp.test(updatedValue)) {
+	} else if (!nameRegExp.test(updatedValue) || updatedValue.length > 64) {
```

Comment:
Create a const above say 
const MAX_STRING_CHARS = 255;
```suggestion
	} else if (!nameRegExp.test(updatedValue) || updatedValue.length > MAX_STRING_CHARS) {
```